apps/analytics/tsconfig.json

```diff
@@ -1,12 +1,8 @@
 {
   "extends": "tsconfig/nextjs.json",
   "compilerOptions": {
-    "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "target": "ES6",
```

Comment:
what is the reason for this change?

Reply:
Sorry for not putting the docs ref: https://github.com/drizzle-team/drizzle-orm/issues/803 it's an issue from Drizzle